File: src/main/java/com/flexingstudios/FlexingNetwork/api/Attributes.java
package com.flexingstudios.FlexingNetwork.api;

import org.bukkit.ChatColor;

public enum Attributes {
    NEW(ChatColor.RED, "НОВОЕ"),
    UPDATE(ChatColor.AQUA, "ОБНОВЛЕНИЕ"),
    UPDATED(ChatColor.GREEN, "ОБНОВЛЕНО"),
    DISCOUNT(ChatColor.GOLD, "СКИДКА"),
    RELEASED(ChatColor.YELLOW, "ВЫПУЩЕНА"),
    FEATURED(ChatColor.GREEN, "ПОПУЛЯРНОЕ")
    ;

    private final ChatColor color;
    private final String tag;

    Attributes(ChatColor color, String tag) {
        this.color = color;
        this.tag = tag;
    }

    public ChatColor getColor() {
        return color;
    }

    public String getTag() {
        return tag;
    }
}

File: src/main/kotlin/com/flexingstudios/flexingnetwork/api/util/Attributes.kt
package com.flexingstudios.flexingnetwork.api.util

import net.md_5.bungee.api.ChatColor

enum class Attributes(
    val color: ChatColor,
    val tag: String,
) {
    NEW(ChatColor.RED, "НОВОЕ"),
    UPDATE(ChatColor.AQUA, "ОБНОВЛЕНИЕ"),
    UPDATED(ChatColor.GREEN, "ОБНОВЛЕНО"),
    DISCOUNT(ChatColor.GOLD, "СКИДКА"),
    RELEASED(ChatColor.YELLOW, "ВЫПУЩЕНА"),
    FEATURED(ChatColor.GREEN, "ПОПУЛЯРНОЕ"),
    ;

    fun addAttribute(color: ChatColor, tag: String): String {
        return Utilities.colored("$color $tag")
    }
}